### ISSUE TITLE
Visual and Keyword search

### ISSUE THREAD

**Issue by spykoulouris** [ArtResearch/vision-app#98]
I was wondering whether in the [visual search](https://artresearch.net/resource/VisualSearch) we can combine a visual + keyword search.

That is search for similar images whose records include a specific word. Do you think something like that will give additional value to our project @lklic @lwruby ?

**Comment by lwruby** [comment]
I think it would be excellent. Is it possible Lukas?

Louisa

**Comment by lklic** [comment]
Yes this functionality is currently on the backburner. I created an issue for it a while ago:
https://github.com/ArtResearch/artresearch.net/issues/242

That issue relates to metadata only. Once we integrate visual search into the main platform (not on vision which is only meant for artwork matching via images), we should be able to combine this with metadata search to perform matches. I already have a plan for how to do this, but the implementation is a major enhancement, so we need to wait on our next round of funding to be able to support this.

I will keep this issue open for now and add a link to it from the other issue.
